Chatkit/View Models/TypingUsersViewModel.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+/// A view model which provides a text representations of typing users.
+public class TypingUsersViewModel {
+    
+    // MARK: - Properties
+    
+    /// The text representations of typing users.
+    public private(set) var value: String?
+    
+    /// The typing users provider used as the source of data.
+    public let provider: TypingUsersProvider
+    
+    /// The placeholder used when a user doeas not have a value set for the name property.
+    public let userNamePlaceholder: String
```

Comment:
Surely this is private? What's the benefit in exposing it?

Reply:
Please see the comment above.

Reply:
After the discussion the property is now marked as `private`.